tools/cli/help.txt

```diff
@@ -1,19 +1,25 @@
 >>>
 (the first line of each entry is the summary for autogenerated command lists)
-Usage: meteor [--release <release>] [--help] <command> [args]
-       meteor help <command>
+Usage: meteor [--release <release>] [--help] <built-in command> [args]
+       meteor help <built-in command>
+       meteor <external command> [args]       
        meteor [--version] [--arch]
+       
 
 With no arguments, 'meteor' runs the project in the current
 directory in local development mode. You can run it from the root
 directory of the project or from any subdirectory.
 
 Use 'meteor create <path>' to create a new Meteor project.
 
-Commands:
+Built-in Commands:
 {{commands}}
 
-See 'meteor help <command>' for details on a command.
+See 'meteor help <built-in command>' for details on a command.
+
+You can use 'meteor <external command> [args]' to run a binary command
+installed previously with 'meteor npm install -g <package with command>',
+which will store the package under the global meteor installation folder.
```

Comment:
What if we changed this to say:

```
External commands (e.g. `yarn`) can be installed within Meteor itself
and used within all apps using the same version:

    meteor npm install --global <name>

And used with:

    meteor <external command>
```

Reasons being:

 * I think it's important to use `--global` instead of `-g` for the sake of clarity
 * I think`yarn` as an example is helpful since it's probably the number one use-case
 * Even though NPM tends to call them `binary` commands, they aren't really "binaries".